athena/modules/modeling/module_modeling_llm/module_modeling_llm/core/generate_suggestions.py

```
@@ -5,16 +5,21 @@
 from athena import emit_meta
 from module_modeling_llm.config import BasicApproachConfig
 from module_modeling_llm.models.assessment_model import AssessmentModel
-from module_modeling_llm.prompts.apollon_format_description import apollon_format_description
+from module_modeling_llm.prompts.apollon_format_description import (
+    apollon_format_description,
+)
 from llm_core.utils.predict_and_parse import predict_and_parse
 from module_modeling_llm.prompts.graded_feedback_prompt import GradedFeedbackInputs
 from module_modeling_llm.models.exercise_model import ExerciseModel
+from langchain_core.prompts import ChatPromptTemplate
```

Ruff (0.11.9)

Redefinition of unused `ChatPromptTemplate` from line 3

Remove definition: `ChatPromptTemplate`

(F811)